lib/reporters/html-cov.js

```diff
@@ -17,25 +18,31 @@ exports = module.exports = HTMLCov;
  *
  * @api public
  * @param {Runner} runner
+ * @param {Object} options Runner optional options
  */
-function HTMLCov(runner) {
+function HTMLCov(runner, options) {
+  JSONCov.call(this, runner, options, false);
+
   var jade = require('jade');
   var file = join(__dirname, '/templates/coverage.jade');
   var str = readFileSync(file, 'utf8');
   var fn = jade.compile(str, { filename: file });
   var self = this;
 
-  JSONCov.call(this, runner, false);
-
   runner.on('end', function() {
-    process.stdout.write(fn({
+    self.write(fn({
       cov: self.cov,
       coverageClass: coverageClass
     }));
   });
 }
 
 /**
+ * Inherit from `JSONCov.prototype`.
+ */
+inherits(HTMLCov, JSONCov);
```

Comment:
I found the line:
```javascript
function HTMLCov(runner, options) {
  JSONCov.call(this, runner, options, false);
```

Confusing at first, since `HTMLCov` it didn't inherit `JSONCov`. I understand is completely valid code and that is being used to reuse the initialization code, but I found odd to call the constructor function of another object while not inheriting, and would find it double odd to call it while inheriting something else :). If you want, I'll change it though.

About the comment, just followed what looked as a common pattern in the rest of the codebase (most times `inherits` is used, a comment with that form appears), I can remove it if you prefer.